src/app/config/confparse.c

```diff
@@ -642,7 +642,7 @@ config_dump(const config_format_t *fmt, const void *default_options,
       continue;
     }
     /* Don't save 'hidden' control variables. */
```

Comment:
This comment is also confusing, we need to distinguish between "invisible" (no read and no dump) and "hidden" (no dump).

Reply:
Agreed. Must solve now, or should solve before we call this work finished?

Reply:
Should solve before we call #29211 finished.